modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,136 @@
+
+[NOTE]
+--
+To prevent service disruptions, identify any accounts that are used to programmatically call Anypoint Platform, or _service accounts_. You must either configure these calls using the xref:connected-apps-overview.adoc[Connected Apps] feature or add these accounts to the *Exempt Accounts* section when configuring multi-factor authentication requirements. 
+--
+
+To enable multi-factor authentication for all user accounts:
+
+. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+. Navigate to *Access Management*.
+. In the navigation menu, click *Multi-factor auth*.
+. Select *Required*.
+. If you want to exempt specific users or service accounts from multi-factor authentication, add them to the *Exempt Accounts* section.
+. Click *Save*. +
+All nonexempt users in your organization are now required to configure a verification method when they log in to Anypoint Platform.
+
+=== Exempt Users from Multi-factor Authentication
+
+Service users or accounts that make programmatic calls to Anypoint Platform should be exempt from multi-factor authentication because they cannot provide verification aside from a username and password.
+
+It is best to use the xref:connected-apps-overview.adoc[Connected Apps] feature to authorize apps to make programmatic calls to Anypoint Platform instead of using individual user accounts. Using the Connected Apps feature provides more security and control than using user accounts.
+
+To exempt individual accounts from multi-factor authentication:
+
+. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+. Navigate to *Access Management*.
+. In the navigation menu, click *Multi-factor Auth*.
+. In the *Exempt Accounts* section, add the accounts that you want to exempt. +
+Only users who do not use single sign-on (SSO) appear in this list.
+. Click *Save*. +
+The exempt user accounts appear in the *Exempt Accounts* section.
+
+You can later remove user accounts from the *Exempt Accounts* list by clicking *X* next to their account names.
+
+== Enable Multi-factor Authentication on Your User Account
+
+You can enable multi-factor authentication on your user account even if your organization does not require it. You can configure more than one verification method for your account:
+
+. Log in to Anypoint Platform.
+. Navigate to your user profile.
+. Click *Configure multi-factor authentication (MFA)*. +
+The *Manage Your Verification Methods* interface appears.
+. Next to the verification method you want to configure, click *Add*.
+. Follow the instructions for your preferred verification method.
+
+== Modify or Remove Multi-factor Authentication on Individual User Accounts
```

Comment:
```suggestion
== Modify or Remove Multi-Factor Authentication on Individual User Accounts
```

Reply:
Are these individual users in an org or outside of an org?
Is the admin doing this or the user?
```suggestion
== Modify or Remove Multi-Factor Authentication on Your Account?
```